Controller/CRUDController.php

```diff
@@ -120,12 +120,13 @@ public function listAction()
      * Execute a batch delete.
      *
      * @param ProxyQueryInterface $query
+     * @param Request $request
      *
      * @return RedirectResponse
      *
      * @throws AccessDeniedException If access is not granted
      */
-    public function batchActionDelete(ProxyQueryInterface $query)
+    public function batchActionDelete(ProxyQueryInterface $query, Request $request)
```

Comment:
@Soullivaneuh No : https://3v4l.org/aRfPr , because it breaks the LSP (you can't call the child class with a request argument)